RecoHI/HiEvtPlaneAlgos/python/RecoHiEvtPlane_EventContent_cff.py

```diff
@@ -10,8 +10,7 @@
 
 RecoHiEvtPlaneAOD = cms.PSet(
     outputCommands = cms.untracked.vstring('keep recoEvtPlanes_hiEvtPlane_*_*',
-                                           'keep ZDCRecHitsSorted_zdcreco_*_*',
```

Comment:
why did you drop the hits?
the keep statement should be here (despite the fact that currently these hits don't make much sense for run2 data/data unpacked into castorDigis)